commands/shooter/trackhood.py

```diff
@@ -0,0 +1,24 @@
+from commands2 import CommandBase
+from wpilib import SmartDashboard
+from wpimath.geometry import Rotation2d
+
+from subsystems.shootingsubsystem import ShootingSubsystem
+import constants
+
+
+class TrackHood(CommandBase):
```

Comment:
Same thing, why have a separate command for the hood if we have the one with them both combined? We could use one at a time if we wanted on that